src/libraries/Native/Unix/System.Globalization.Native/pal_icushim_static.c

```diff
@@ -0,0 +1,81 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+
+#include <stdlib.h>
+#include <stdio.h>
+#include "pal_icushim_internal.h"
+#include "pal_icushim.h"
+#include <unicode/putil.h>
+#include <unicode/uversion.h>
+#include <unicode/localpointer.h>
+#include <unicode/utrace.h>
+
+static void log_icu_error (const char * name, UErrorCode status) {
```

Comment:
convention in the rest of the pal code seems to be braces on a new line and no space after method name.
```suggestion
static void log_icu_error(const char * name, UErrorCode status)
{
```